does not build with Kernel 5.4

while current HEAD builds fine up to Kernel 5.3.x it fails with Kernel 5.4 (which just went into Ubuntu 18.04 Bionic as their HWE-Kernel):

[make.log](https://github.com/freifunk-gluon/batman-adv-legacy/files/4958656/make.log)

```
..
CC [M]  /var/lib/dkms/batman-adv/2013.4.0/build/soft-interface.o
/var/lib/dkms/batman-adv/2013.4.0/build/soft-interface.c: In function ‘batadv_interface_rx’:
/var/lib/dkms/batman-adv/2013.4.0/build/soft-interface.c:319:2: error: implicit declaration of function ‘nf_reset’; did you mean ‘bio_reset’? [-Werror=implicit-function-declaration]
  nf_reset(skb);
  ^~~~~~~~
  bio_reset
```

--- a/compat.h
+++ b/compat.h
@@ -709,4 +709,10 @@ batadv_genl_unregister_family(struct batadv_genl_family *family)
 
 #endif /* < KERNEL_VERSION(5, 2, 0) */
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 4, 0)
+
+#define nf_reset_ct nf_reset
+
+#endif /* < KERNEL_VERSION(5, 4, 0) */
+
 #endif /* _NET_BATMAN_ADV_COMPAT_H_ */

--- a/soft-interface.c
+++ b/soft-interface.c
@@ -316,7 +316,7 @@ void batadv_interface_rx(struct net_device *soft_iface,
 	/* clean the netfilter state now that the batman-adv header has been
 	 * removed
 	 */
-	nf_reset(skb);
+	nf_reset_ct(skb);
 
 	ethhdr = eth_hdr(skb);
 

